### PR TITLE
add git config fix for build CI/CD git issue

### PR DESCRIPTION
**Pull request by dario-piotrowicz**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -184,6 +184,7 @@ jobs:
           apiToken: ${{ secrets.CF_API_TOKEN }}
           preCommands: |
             echo "*** pre commands ***"
+            git config --global --add safe.directory /github/workspace
```
Comment (petebacondarwin) [inline .github/workflows/deploy.yml:187]: Is `/github/workspace` the actual path that we use or just some boilerplate that was in the error message?

**Review by petebacondarwin — APPROVED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (dario-piotrowicz) [reply .github/workflows/deploy.yml:187]: > Is /github/workspace the actual path that we use or just some boilerplate that was in the error message?

ah... yeah, I'm not sure 😓

we can merge and give it a try?

or I could use `${{ github.workspace }}` instead:
[Screenshot 2023-01-12 at 11 14 54](https://docs.github.com/en/actions/learn-github-actions/contexts)
?

Comment (dario-piotrowicz) [reply .github/workflows/deploy.yml:187]: I think @xtuc suggested that it is coming from wrangler?
https://github.com/cloudflare/wrangler-action/blob/master/Dockerfile#L3

Comment (xtuc) [reply .github/workflows/deploy.yml:187]: can't we find another way to getting the git hash?

Comment (dario-piotrowicz) [reply .github/workflows/deploy.yml:187]: sure, @petebacondarwin suggested that in the chat, let me have a look...

**Comment by dario-piotrowicz** [comment]
Closing this PR in favour of https://github.com/cloudflare/wildebeest/pull/93

**Pull request closed without merging**